components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -20,6 +20,9 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.json.JSONObject;
+import org.slf4j.MDC;
```

Comment:
Is this import needed?

Reply:
it's used in https://github.com/wso2-extensions/identity-governance/pull/452/files/db3db432beb65c5e6174862949a01252d2f26d94#diff-d9563798eea549a0b0451cfb68c0a7a588b10038ae656cd6fc29ef12df4a5745R528